lib/supavisor/client_handler.ex

```diff
@@ -194,7 +194,7 @@ defmodule Supavisor.ClientHandler do
            {:next_event, :internal, event}}
         else
           reason = "Invalid format for user or db_name"
-          Logger.error("ClientHandler: #{inspect(reason)}")
+          Logger.error("ClientHandler: #{inspect(reason)} #{inspect({user, db_name})}")
```

Comment:
We need to start using structured logging…

Reply:
we have the username and database name in the log metadata, but combining them here into a single string would make it more clear

Reply:
Also put them in the metadata